functions/Restore-DbaBackup.ps1

```diff
@@ -0,0 +1,215 @@
+function Restore-DbaBackup
```

Comment:
@ctrlbold  - as all the helper functions are now in dev, are you ok if I pull them into this branch for changes? Or would you rather I still keep things split?